config_utilities/src/commandline.cpp

```diff
@@ -289,7 +298,7 @@ YAML::Node nodeFromFileEntry(const CliParser::Entry& entry) {
   std::filesystem::path file(filepath);
   if (!fs::exists(file)) {
     std::stringstream ss;
-    ss << "File " << file << " does not exist!";
+    ss << "File '" << file << "' does not exist!";
```

Comment:
(minor) the streaming operator for filepaths adds quotes by default (I think you end up with the literal `'"/some/filepath"'` unless you do `file.string()`